.github/workflows/test.yml

```diff
@@ -3,6 +3,7 @@ run-name: 🧪 Tests (${{ github.event_name }})
 
 on:
   workflow_dispatch:
+  workflow_call:
```

Comment:
is this change intended? seems like it makes `test.yml` callable in another workflow but I'm not seeing it being called anywhere

Reply:
No, good catch. Snipped.